tests/MCPClient/test_antivirus.py

```diff
@@ -72,10 +73,17 @@ def __init__(self, should_except=False, passed=False):
     def scan(self, path):
         if self.should_except:
             raise Exception("Something really bad happened!")
-        return self.passed, None, None
+        result = ScanResult(filename=path, state="OK", details="details", err=None)
+        mock.patch.object(
+            result.__class__,
+            "passed",
+            new_callable=mock.PropertyMock,
+            return_value=self.passed,
+        ).start()
```

Comment:
Is this patch stopped implicitly?

Reply:
I don't really know what I'm doing, but I've tried returning a a `ScanResultMock` instead and it seems to work.